src/index_runner/event_loop.py

```diff
@@ -65,46 +62,30 @@ def start_loop(
             continue
         val = msg.value().decode('utf-8')
         try:
-            msg = json.loads(val)
+            val_json = json.loads(val)
         except ValueError as err:
             logger.error(f'JSON parsing error: {err}')
             logger.error(f'Message content: {val}')
             consumer.commit()
             continue
-        logger.info(f'Received event: {msg}')
+        logger.info(f'Received event: {val_json}')
         start = time.time()
         try:
-            message_handler(msg)
-            # Move the offset for our partition
-            consumer.commit()
-            on_success(msg)
-            logger.info(f"Handled {msg['evtype']} message in {time.time() - start}s")
+            message_handler(val_json)
         except Exception as err:
             logger.error(f'Error processing message: {err.__class__.__name__} {err}')
             logger.error(traceback.format_exc())
             # Save this error and message to a topic in Elasticsearch
-            on_failure(msg, err)
-
-
-# might make sense to move this into the config module
-def _fetch_latest_config_tag():
```

Comment:
are the update functions for fetching the newest config already in the config file?

Reply:
so I changed it so we just configure the URL of the config file directly, such as `https://raw.githubusercontent.com/kbase/index_runner/develop/spec/config.yaml`. I'm not doing the release url stuff anymore because I think this achieves the same thing while being simpler. You could link a specific branch, commit, release, or even a github gist if you want.

To get an automatic update, the config just needs to re-request that url and doesn't need to mess with the release API